bot/exts/moderation/voice_gate.py

```diff
@@ -53,6 +67,13 @@ async def voice_verify(self, ctx: Context, *_) -> None:
         - You must not be actively banned from using our voice channels
         - You must have been active for over a certain number of 10-minute blocks
         """
+        # If user has received a ping in voice_verification, delete the message
+        if message_id := await self.redis_cache.get(ctx.author.id, NO_MSG):
+            with suppress(discord.NotFound):
+                ping_message = await ctx.channel.fetch_message(message_id)
+                await ping_message.delete()
```

Comment:
This could be simplified by directly using the http method if you want:
```python
await self.bot.http.delete_message(channel_id, message_id)
```
Then we'd only be making one API call as we don't have to fetch the message first.